modules/images/image-loading-optimization/storage/data.php

```diff
@@ -186,8 +193,17 @@ static function ( $a, $b ) {
  *  3. 481-576 (phablets)
  *  4. >576 (desktop)
  *
+ * The default breakpoints are reused from Gutenberg where the _breakpoints.scss file includes these variables:
+ *
+ *     $break-medium: 782px; // adminbar goes big
```

Comment:
Curious about the admin bar comment: Is that relevant? Since we should exclude the admin bar from the considerations here, I would say if this breakpoint is only to cater for that, we would rather need to find a different solution.

In other words, if 782px is a real breakpoint also for the regular frontend content, let's use it. But if it's only relevant as the admin bar breakpoint, we shouldn't need to capture LCP image data for that.